phptravels website/test_hotel_subpage.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+import time
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.common.keys import Keys
+
+# 1. Log in
+# 2. Navigate to "Hotels" subpage
+# 3. Test different boundary values in the search element
+
+
+def room_fillout(driver, get_xpaths, options):
+    room_choice = driver.find_element(By.XPATH, get_xpaths["subpages"]["room_choice"])
+    room_choice.click()
+    WebDriverWait(driver, 10).until(EC.presence_of_element_located((By.XPATH, get_xpaths["subpages"]["room_inc"])))
+
+    for key, value in options.items():
+        driver.find_element(By.XPATH, "//input[@id='" + key + "']").clear()
+        driver.find_element(By.XPATH, "//input[@id='" + key + "']").send_keys(options[key])
+
+
+@pytest.mark.parametrize("rooms", [-1, "abc", 0, 2, 20])
+@pytest.mark.parametrize("adults", [-1, "abc", 0, 2, 5000])
+@pytest.mark.parametrize("childs", [-1, "abc", 2, 5000])
+def test_boundary(get_login, get_settings, get_xpaths, get_cred, rooms, adults, childs):
+
+    driver = get_login
+
+    time.sleep(2)
```

Comment:
let's try not to hard code time.sleep() unless it's completely necesssary

Reply:
Removed